nixos/modules/security/sudo.nix

```diff
@@ -207,10 +209,8 @@ in
       ''
         # Don't edit this file. Set the NixOS options ‘security.sudo.configFile’
         # or ‘security.sudo.extraRules’ instead.
-
-        # Keep SSH_AUTH_SOCK so that pam_ssh_agent_auth.so can do its magic.
-        Defaults env_keep+=SSH_AUTH_SOCK
-
+      ''
```

Comment:
Just noticed this: is this deliberate or a rebase/amend issue?

Reply:
Ah, I looked at it only in the context of 19e1420e138b5a89f00eb3d7adbdf99f97ad4d57 which doesn't have that `pipe.cfgExtraRules`, it's just two multiline strings in a row. It's a cosmetic thing in an intermediate commit, I don't care too much :)